### PR TITLE
Fix logging in workers

### PR DESCRIPTION
**Pull request by hawkowl**
This also adds a worker blacklist.

### CODE REVIEW

File: worker-blacklist [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# This file serves as a blacklist for SyTest tests that we expect will fail in
```
Comment (richvdh) [inline worker-blacklist]: could these go somewhere other than the root of the repository? There's enough clutter there already.

File: changelog.d/5729.bugfix [hunk 2]
```diff
@@ -0,0 +1 @@
+Fix worker log config not being set correctly.
```
Comment (richvdh) [inline changelog.d/5729.bugfix]: probably wants merging with the PR that broke it rather than being a new changelog entry?

Comment (hawkowl) [reply changelog.d/5729.bugfix]: err yes, done

File: worker-blacklist (hunk 1 above)
Comment (hawkowl) [reply worker-blacklist]: done

**Comment by richvdh** [comment]
It would kinda be nice to separate the blacklist stuff to a separate PR...

**Comment by hawkowl** [comment]
@richvdh I was doing it so that the worker builder went green for real instead of masking a problem :P

**Comment by hawkowl** [comment]
(I can split it out)

File: MANIFEST.in [hunk 3]
```diff
@@ -8,6 +8,7 @@ include demo/demo.tls.dh
 include demo/*.py
 include demo/*.sh
 include sytest-blacklist
+include worker-blacklist
```
Comment (richvdh) [inline MANIFEST.in]: except it's now under `.buildkite` ?

**Comment by richvdh** [comment]
> (I can split it out)

shrug

**Pull request merged**